src/Message/Formatter.php

```diff
@@ -139,15 +142,16 @@ private function defaultFormat(Message $message, array $commonContext): string
      */
     private function getTime(Message $message): string
     {
-        $timestamp = (string) $message->context('time', microtime(true));
+        /** @var float $timestamp */
+        $timestamp = $message->context('time', microtime(true));
```

Comment:
```suggestion
        $timestamp = (string) $message->context('time', microtime(true));
```

Reply:
Psalm shows error without the type declaration
```
PossiblyInvalidCast - src/Message/Formatter.php:146:31 - array{category?: string, memory?: int, time?: float, trace?: list<array{class?: string, file: string, function?: string, line: int, type?: string}>} cannot be cast to string (see https://psalm.dev/190)
        $timestamp = (string) $message->context('time', microtime(true));
```

Reply:
Psalm can not determine type of result, but we known that for time always returns float. I think we can suppress this error.